Dockerfile

```diff
@@ -75,19 +71,30 @@ RUN ./configure --with-python --with-geos=yes \
     && sudo ldconfig
 WORKDIR /
 
+# TODO: iROds 4.2.x is holding us to Debian Buster which is EOL. It also requires libssl1.0.0 which is obsolete
+# we should upgrade to iRods 4.3, Debian Bullseye, etc but this will require at a minimum, changes to our iinit use in HS
```

Comment:
Roadmap? ;)
I think 4.3 upgrade should be on our map for sure! libssl isn't the only security concern that we have while we are held to Buster and it's EOL. Let's I propose that we merge this PR (because without it, this base image doesn't even currently build) and discuss an upgrade to iRods 4.3 and Debian Bullseye during our next meeting!